src/app/pages/network/components/interfaces-card/interfaces-card.component.ts

```diff
@@ -188,7 +199,17 @@ export class InterfacesCardComponent implements OnInit, OnChanges {
 
   private subscribeToUpdates(): void {
     this.networkService.subscribeToInOutUpdates().pipe(untilDestroyed(this)).subscribe((updates) => {
-      this.inOutUpdates = updates;
+      if (!updates) {
+        return;
+      }
+      const newInOutUpdates = _.cloneDeep(this.inOutUpdates);
+      const updatedInterfaces = Object.keys(updates);
+      for (const nic of updatedInterfaces) {
+        newInOutUpdates[nic] = {
```

Comment:
the spread operator { ...updates[nic] } is redundant here since updates[nic] is directly assigned to newInOutUpdates[nic]. You can simplify this to `newInOutUpdates[nic] = updates[nic];`

Reply:
Explained in the separate comment. Let me know what you think.